core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -615,6 +615,16 @@ public boolean decide(String workflowId) {
 
             stateChanged = scheduleTask(workflow, tasksToBeScheduled) || stateChanged;
 
+            if (!outcome.tasksToBeUpdated.isEmpty()) {
```

Comment:
Is this a change coming from upstream? If so it shouldn't be part of this commit PR. If it isn't, I'm pretty sure we shouldn't be modifying WorkflowExecutor in this PR.

Reply:
That is part of bug I that I have opened on another PR #781 . I have removed this file